src/Components/Tabs/FAQ.scss

```diff
@@ -0,0 +1,14 @@
+.ins-c-card {
```

Comment:
i think this is n existing pf class name, might want to rename to something else, more specific to the app

BUT we should probably use things that are similar to whats in cardcollection.scss, maybe bubble that styling up to app.scss, make it common across?